### PR TITLE
398 quotation marks

### LINKED ISSUE #398: Replace German quotation marks by English quotation marks
Example from the [DELETE](https://opensource.zalando.com/restful-api-guidelines/#delete) article:

> The semantic is best described as »please delete the resource identified by the URL«.

Those are German styled quotation marks. If this documentation is in English, it should read:

> The semantic is best described as "please delete the resource identified by the URL".

## Task

Search in all of the documentation and fix all of them, not only the example provided here.

### PR DESCRIPTION
**Pull request by ddufourboivin-zalando**
fix #398 

Two commits:
1. Replace the `»...«` by `"..."`
1. Replace the `"..."` by `“...”` everywhere

Options:
1. We keep the ASCII quotes `"..."`
    - Pros
      - Easier to type using any keymap
      - Probably what is already used when pasting text from other sources
    - Cons
      - Not typographically accurate
      - There is already a mix of `"..."` and `“...”` in this documentation
1. We switch everything to `“...“`
    - Pros
      - Typographically accurate
      - Align the entire documentation to one quotation standard
    - Cons
      - Bigger diff (not really a problem)
      - Hard to maintain (people will probably have the reflex of using `"..."` in the future).

I did it in 2 different commits, which will be useful if we go with the 1st option (rollback the 2nd).

Let's discuss!

### CODE REVIEW

**Comment by whiskeysierra** [comment]
Authors of the guidelines write in markdown and markdown is code. We very much need to stick to the obvious choice of `"..."`. What I could imagine is some post processing that rewrites quotes before publication.

**Comment by ddufourboivin-zalando** [comment]
I personally do not have an opinion. Or if one, stick to `"..."`. This task started as _getting rid of the German-styled quotes_ and after a comment from @ePaul , went to _apply correct typography_.

I would vote for `"..."` because it is easier to maintain. But then, we need to stick to this styling and not have an hybrid, as it is at the moment.

File: chapters/events.adoc [hunk 1]
```diff
@@ -286,8 +286,9 @@ link:#event-metadata[`metadata`] field being reserved for standard
 information (the contents of link:#event-metadata[`metadata`] are
 described further down in this section).
 
-In the example fragment below, the reserved `metadata` field is shown
-with fields “a” and “b” being defined as part of the custom schema:
+In the example fragment below, the reserved link:#event-metadata[`metadata`]
```
Comment (maxim-tschumak) [inline chapters/events.adoc]: <<event-metadata,metadata>> should also work

Comment (ddufourboivin-zalando) [reply chapters/events.adoc]: @maxim-tschumak : https://github.com/zalando/restful-api-guidelines/pull/404

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by tkrop** [comment]
@ddufourboivin-zalando Thanks for the work. However, most of the `"..."` are in reality for emphasizing terms not for quoting. I would prefer these to use `_..._` instead (or `*...*` in strong cases). When it comes to quoting, it should always use `_"..."_` to keep it simple in code, even if I like the German quotes.

**Comment by ddufourboivin-zalando** [comment]
@tkrop : I will rework this branch to use `"..."` everywhere (basically reversing the commit where I change them to `“...”`). Let's align on the character to use for quoting first and discuss the use of _quotation_ vs _emphasis_ in a next discussion (my proposal).

**Comment by ddufourboivin-zalando** [comment]
# Next question ...

Which brings us to the next consideration.

*Should we align the existing `“...”` to the decided `"..."` or leave them be?*

Please vote by 👍 or 👎 this comment (smiley in up-right corner).

**Comment by ePaul** [comment]
@whiskeysierra 
> Authors of the guidelines write in markdown and markdown is code. We very much need to stick to the obvious choice of "...".

I don't really follow that argumentation. Is it »Code should only ever use ASCII«?

**Comment by whiskeysierra** [comment]
> I don't really follow that argument. Is it »Code should only ever use ASCII«?
> (I'm fine with using ASCII codes, I just want to understand the argument here.)

Yes.

**Comment by tkrop** [comment]
@ePaul not entirely. It is although that this is a English document, where we should use punctuation marks (here quotation marks) as usually used in this language :).

**Comment by tkrop** [comment]
:+1:

**Comment by ddufourboivin-zalando** [comment]
Since it is a document, we _should_ be using the right quotation marks `“...”`, but I also think that it would be difficult to maintain consistency because we'll forever have the reflex of using ASCII/coding quotations `"..."` when we right. So I am completely good with this decision of using the ASCII ones.

There will be another discussion (initiated by @tkrop ) about using proper _emphasis_ instead of "quotations" in this documentation where it applies (most of the actual quoted text should indeed be emphasised instead). But this is out of scope here.

I will be pushing a last time with the decided "quotation marks" and then we can close this topic.

**Comment by ddufourboivin-zalando** [comment]
Ready for re-review.

@whiskeysierra @tkrop @maxim-tschumak @epaul

**Pull request merged**